src/pages/register-a-beacon/activity.tsx

```diff
@@ -30,7 +40,7 @@ const definePageForm = ({
       [
         {
           dependsOn: "activity",
-          meetingCondition: (value) => value === MaritimePleasureVessel.OTHER,
+          meetingCondition: (value) => value === MaritimePleasureType.OTHER,
```

Comment:
Not sure the nicest way to handle this, since right now both `Maritime Pleasure` and `Maritime Commercial` have an _Other_. 😬 

Could we just have one big enum for Activity Type? Or is that no bueno? 🤔

Reply:
@Ninamma Agree with the second point when pairing with @zackads and @NyWeb that we will have just one `Activity` enum.

For when we come to doing the other pages (aircraft, pleasure/commercial), we could look to have an object lookup that specifies valid combinations of activities based on what the user has selected.